### PR TITLE
Created Ethiopa legislation preprocessing script

### LINKED ISSUE #5: Develop preprocessing script for legislation from Ethiopia

### PR DESCRIPTION
**Pull request by Skyrolon**
Closes #5

### CODE REVIEW

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
...ronment so as to affect its beneficial use adversely, or [b] produces toxic substances, diseases, objectionable odour, radioactivity, noise, vibration, heat, or any other phenomenon that is hazardous or potentially hazardous to human health or to other living things; [8] "Project" mans any new development activity under any category listed in any directive issued pursuant to this Proclamation, major expansion or alteration or any existing undertaking, or any resumption of work that had been discontinued; [9] "Proponent" means any organ of government if in the public sector or any person if in the private sector that initiates a project; [10] "Public instrument" means a policy, a strategy, a programme, a law or an international agreement; [11] "Region" means any of those parts of Ethiopia specified as regions under Article 47[1] of the Constitution of the Federal Democratic Republic of Ethiopia and, for the purpose of this Proclamation, includes the Addis Ababa and Dire Dawa Administrations; [12] "Regional Environmental Agency" means any regional government organ entrusted by that Region with a responsibility for the protection or regulation of the environment and natural resources.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please replace `to, land atmosphere, whether` with `to land, atmosphere, weather`. Also, please replace `"Project" mans` with `"Project" means`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Review of Environmental Impact Study Report
+(9) [1] An environmental impact study report that is submitted to the Authority or the relevant regional environmental agency for review shall include a brief statement summarizing the study in non-technical terms as well as indicating the completeness and accuracy of the information given in the study report. [2] The Authority or the relevant regional environmental agency shall, after evaluating an environmental impact study report by taking into account any public comments and expert opinions, within 15 working days: [a] approve the project without conditions and issue authorization if it is convinced that the project will not cause negative impacts; [b] approve the project and issue authorization with conditions that must be fulfilled in order to eliminate or reduce adverse impacts or reduce adverse impacts to insignificance if it is convinced that the negative impacts can be effectively countered, or [c] refuse implementation of the project if it is convinced that the negative impact cannot be satisfactorily avoided.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please replace `countered, or [c]` with `countered; or [c]`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Validity of Approved Environmental Impact Study Report
+(10) [1] The Authorization of an environmental impact study report shall expire if the project has not been implemented according to the time frame set during its authorization. [2] Any proponent who wishes to challenge the appropriateness of the provision of Sub Article [1] of this Article, to his project, may submit an application to that effect to the Authority or the relevant regional environmental agency, as may be appropriate. [3] Within 30 days from the receipt of an application pursuant to Sub Article [2] of this Article, the Authority or the relevant regional environmental agency shall, unless special circumstance so dictate, decide whether to extend the validity of the report or to order the revision or the redoing of the environmental impact assessment.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please replace `circumstance` with `circumstances`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 4]
```diff
@@ -0,0 +1,82 @@
+PART ONE
+
+Short Title
+(1) This Proclamation may be cited as the "Environmental Impact Assessment Proclamation No. 299/2002."
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt:4]: I'll review this, but please recall that we no longer wish to preprocess any EIA-specific legislation. We will later need to replace the preprocessing script in this pull request with one for Ethiopia's environment code, if such a code exists.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Validity of Approved Environmental Impact Study Report
+(10) [1] The Authorization of an environmental impact study report shall expire if the project has not been implemented according to the time frame set during its authorization. [2] Any proponent who wishes to challenge the appropriateness of the provision of Sub Article [1] of this Article, to his project, may submit an application to that effect to the Authority or the relevant regional environmental agency, as may be appropriate. [3] Within 30 days from the receipt of an application pursuant to Sub Article [2] of this Article, the Authority or the relevant regional environmental agency shall, unless special circumstance so dictate, decide whether to extend the validity of the report or to order the revision or the redoing of the environmental impact assessment. 
+
+PART FOUR 
+
+Occurrence of New Circumstance
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please replace `Circumstance` with `Circumstances`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+(11) If an unforeseen fact of serious implication is realized after the submission of an environmental impact study report, the Authority or the relevant regional environmental agency may, as may be appropriate, order the environmental impact assessment to be revised or to be redone in order to address the implication.
+
+Implementation Monitoring
+(12) [1] The Authority or the relevant regional environmental agency shall monitor the implementation of an authorized project in order to evaluate compliance with all commitments made by, and obligations imposed on the proponent during authorization. [2] When the proponent fails to implement the authorized project in compliance with the commitments he entered into or obligations imposed upon him, the Authority or the relevant regional environmental agency may order him to undertake specified rectification measure. [3] Any other authorizing or licensing agency shall, in tandem with the Authority's or the relevant regional environmental agency's decision to suspend or cancel any authorization to implement a project, suspend or cancel the license it may have issued in favor of the project.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please replace `rectification measure` with `rectification measures`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+Environmental Impact Assessment of public Instrument
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please capitalize `public`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+Jurisdiction
+(14) [1] The Authority shall be responsible for the evaluation of an environmental impact study report and the monitoring of its implementation when the project is subject to licensing, execution or supervision by a federal agency or when it is likely to produce trans-regional impact. [2] The regional environmental agency in each region shall be responsible for the evaluation and authorization or any environmental impact study report and the monitoring of its implementation if the project is not subject to licensing, execution and supervision by a federal agency and if it is unlikely to produce trans-regional impact. 
+
+PART FIVE
+
+Public participation
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please capitalize `participation`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+PART SEVEN - MISCELLANEOUS PROVISIONS
+
+Power to Issue regulations
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: Please capitalize `regulations`.

File: raw_data/preprocessed/africa/ethiopa_english.txt [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+(20) The Authority may issue directives necessary for the effective implementation of this Proclamation.
+
+Duty to Cooperate
+(21) Any person shall have the duty to cooperate in the implementation of this Proclamation.
+
+Laws
+(22) Any law or practice inconsistent with this Proclamation is inapplicable regarding matters provided herein.
+
+Effective Date
+(23) This Proclamation shall come into force as of the 3rd day of December, 2002. Done at Addis Ababa, this 3rd day of December, 2002.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/ethiopa_english.txt]: I doubt that the second sentence (i.e. `Done at ...`) is part of the article. In all of the other legislation that I have reviewed, that sentence has been associated with the signatures.

**Review by danpdraper — CHANGES_REQUESTED** [review]

File: raw_data/preprocessed/africa/ethiopa_english.txt (hunk 10 above)
Comment (Skyrolon) [reply raw_data/preprocessed/africa/ethiopa_english.txt]: Good point, I have removed it.

File: raw_data/preprocessed/africa/ethiopa_english.txt (hunk 4 above)
Comment (Skyrolon) [reply raw_data/preprocessed/africa/ethiopa_english.txt:4]: This slipped my mind, thank you for the reminder.

**Comment by danpdraper** [comment]
@Skyrolon closed this pull request because it concerned EIA-specific legislation from Ethiopia, not Ethiopia's environment code (if such a code exists).

**Comment by danpdraper** [comment]
Per my comments in cell T26 of the 'Adoption' sheet of the spreadsheet, it does not appear as if Ethiopia has enacted an environment code. @Skyrolon - Please create a new pull request encapsulating the changes that were encapsulated in this pull request. We cannot re-open this pull request because I deleted the issue-5 branch. If you no longer have the issue-5 branch locally, you can checkout this pull request (see the 'Code' dropdown at the top right of this page) instead of recreating the changes from scratch.

**Pull request closed without merging**